src/libs/auth.js

```diff
@@ -110,7 +110,7 @@ export const initializeAuth = _.memoize(async () => {
         fenceStatus: isSignedIn ? state.fenceStatus : {},
         // Load whether a user has input a cookie acceptance in a previous session on this system,
         // or whether they input cookie acceptance previously in this session
-        cookiesAccepted: isSignedIn ? state.cookiesAccepted || getLocalPrefForUserId(user.getId(), cookiesAcceptedKey) : undefined,
+        cookiesAccepted: isSignedIn ? state.cookiesAccepted || getLocalPrefForUserId(user.getId(), cookiesAcceptedKey) : false,
```

Comment:
Just so I understand, the approach here is to wait until GAPI responds with the users sign in credentials, and defers making the decision until after we have a response from GAPI?

My question is (and the guiding philosophy of the original design): what if gapi is slow to respond? During that time, the user could navigate around our open for use portion of the site, and would not necessarily be aware that there are cookies in use. During that period (which admittedly would be small for 99.9% of cases), we would be in violation of GDPR guidelines. We could delay loading the site until after we have a response from GAPI, which would block users from navigating around until we have that response, but in that case, if signin is slow, we would just be having users staring at a blank screen while we wait for GAPI.

Reply:
hm. That's a good point! I would rather not delay loading the site til GAPI, but that means having the cookie warning, and then potentially having it flicker away again, bringing back the original problem.

Right now, we're setting cookiesAccepted from this:
`cookiesAccepted: isSignedIn ? state.cookiesAccepted || getLocalPrefForUserId(...) : false`

What about if we move the check for state before the gapi checks, then have processUser update from localPrefForUserId if it's not set? I think this would still result in a flicker, but hopefully not as frequent as the current scenario

Reply:
That seems fine to me. I don't know how much it will help, since state is ephemeral, so it will reset on refresh. (so the local pref is really the important thing, I think), but it might help some!